.npmignore

```diff
@@ -1,12 +1,9 @@
 .DS_Store
-.eslintcache
 .storybook
-node_modules
```

Comment:
Why'd we add this back into the files sent to npm on publish?

Larger question I'm wondering about: can we switch to the whitelist approach described in link.medium.com/FcQryrm4bV, while we're in here mucking with which files get packed up?

Reply:
Great! I replaced this with the whitelist